spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/archaius/ArchaiusAutoConfiguration.java

```diff
@@ -145,8 +139,7 @@ protected void configureArchaius(ConfigurableEnvironmentConfiguration envConfig)
 			DynamicURLConfiguration defaultURLConfig = new DynamicURLConfiguration();
 			try {
 				config.addConfiguration(defaultURLConfig, URL_CONFIG_NAME);
-			}
-			catch (Throwable ex) {
+			} catch (Throwable ex) {
```

Comment:
Please use the [code formatting conventions](https://github.com/spring-cloud/spring-cloud-netflix/#code-conventions-and-housekeeping-1).